tests/test_base_templates.py

```diff
@@ -95,7 +96,7 @@ def test_codeable_concept_denormalize_filter_creation():
             source AS s,
             UNNEST(s.code_col.coding) AS u (coding)
         WHERE
-            u.coding.system LIKE 'http://snomed.info/sct'
+            REGEXP_LIKE(u.coding.system, 'http:\\/\\/snomed\\.info\\/sct')
```

Comment:
nit: maybe make your eyeballs bleed less with a raw string.

Reply:
i started there - jinja chokes on it, but for this test it could be ok, it just makes updating this for changes a bit fussy.